frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -387,6 +388,10 @@ public String getTorchRunLogDir() {
         return torchrunLogDir;
     }
 
+    public String getMetricsMode() {
+        return getProperty(TS_METRICS_MODE, null);
```

Comment:
default should be log, not null. please also add unit test.

Reply:
It is useful to return `null` in the case where `TS_METRICS_MODE` is not configured and the `mode` configuration is set in the configuration file. Here, we would not want to override the value set in the metrics configuration file with the default value returned by `getMetricsMode`: https://github.com/pytorch/serve/pull/2190/files#diff-c587b142c0b47f58987ec3279409e835e802bb8e27d2202cc504588e21e6f348R76-R79

The case where `mode` is not specified in both the metrics configuration file and in `TS_METRICS_MODE` is handled [here](https://github.com/pytorch/serve/pull/2190/files#diff-c587b142c0b47f58987ec3279409e835e802bb8e27d2202cc504588e21e6f348R81-R83).

So, as per above implementation, the priority order for mode configuration is as follows:
`TS_METRICS_MODE` > `metrics configuration file` > `default mode ("log")`

The default mode value, i.e `"log"` is verified [here](https://github.com/pytorch/serve/pull/2190/files#diff-2f95379a19d5fcc641c241febeb61ec5dcb0207b0224f702df489844d2efa75fR84).

Reply:
To make it simple, let's use ConfigManager as a single place to control the mode. The default value in ConfigManager is log.

the default value can be set in config.property or "TS_xxx" env.